explainability-core/src/main/java/org/kie/trustyai/metrics/language/wer/WordErrorRate.java

```diff
@@ -0,0 +1,36 @@
+package org.kie.trustyai.metrics.language.wer;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.kie.trustyai.metrics.language.utils.AlignedTokenSequences;
+import org.kie.trustyai.metrics.language.utils.TokenSequenceAligner;
+import org.kie.trustyai.metrics.language.utils.TokenSequenceAlignmentCounters;
+
+import opennlp.tools.tokenize.SimpleTokenizer;
+import opennlp.tools.tokenize.Tokenizer;
+
+public class WordErrorRate {
```

Comment:
Hi! Some thoughts that came up. Looking at the [hugging face implementation](https://github.com/huggingface/datasets/blob/main/metrics/wer/wer.py) at the python side they use [jiwer](https://github.com/jitsi/jiwer) which relies on 
some other package in C++. 
a) Do we care about performance comparison? Are there any benchmarks in this area?
b) Would it make sense to use python from the java side for such implementations in general?
c)  Would it make sense if this needs to be implemented in Java to move this to an external project so others could benefit?
d) A nice extension or maybe another totally different idea to explore is use [AI for the error generation and evaluation](https://www.speechmatics.com/company/articles-and-news/the-future-of-word-error-rate).

Reply:
@skonto Regarding c) we are in the process of releasing the TrustyAI core (i.e. the algorithmic core, not the service) as Maven dependency which would allow these methods to be used from any other Java project.
We also have the [Python TrustyAI library](https://github.com/trustyai-explainability/trustyai-explainability-python) ([PyPi](https://pypi.org/project/trustyai/)), which allows for TrustyAI core to be used from the Python ecosystem.

Reply:
Regarding b) We do have a [WIP PR](https://github.com/trustyai-explainability/trustyai-explainability/pull/300) to call Python algorithms from the Java codebase. There are however a few unresolved issues, such as how to solve the inclusion of potentially large Python dependencies in the service's image, and the fact that Python TrustyAI itself bundles Java core (which now would also have Python dependencies).